From 48205c4bd3fda5ad5dd5fadd459e1d6877766d1d Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Mon, 16 Mar 2020 08:28:41 +0200
Subject: [PATCH 1/1] Enforce Solus Defaults:

1. Use transparent background
2. Tweaked percentage for background transparency
3. Set "always" by default for PreserveWorkingDirectory to better handle use of non-bash (e.g. zsh) shells
4. Set ThemeVariant to dark by default
---
 src/org.gnome.Terminal.gschema.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/org.gnome.Terminal.gschema.xml b/src/org.gnome.Terminal.gschema.xml
index 630647c..83d00bf 100644
--- a/src/org.gnome.Terminal.gschema.xml
+++ b/src/org.gnome.Terminal.gschema.xml
@@ -286,7 +286,7 @@
       <description>If true, the command inside the terminal will be launched as a login shell (argv[0] will have a hyphen in front of it).</description>
     </key>
     <key name="preserve-working-directory" enum="org.gnome.Terminal.PreserveWorkingDirectory">
-      <default>'safe'</default>
+      <default>'always'</default>
       <summary>Whether to preserve the working directory when opening a new terminal</summary>
       <description>
         Controls when opening a new terminal from a previous one carries over the working directory of the opening terminal to the new one.
@@ -367,11 +367,11 @@
       <summary>Whether ambiguous-width characters are narrow or wide when using UTF-8 encoding</summary>
     </key>
     <key name="use-transparent-background" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Whether to use a transparent background</summary>
     </key>
     <key name="background-transparency-percent" type="i">
-      <default>50</default>
+      <default>11</default>
       <range min="0" max="100"/>
       <summary>Adjust the amount of transparency</summary>
       <description>A value between 0 and 100, where 0 is opaque and 100 is fully transparent.</description>
@@ -721,7 +721,7 @@
     </key>
 
     <key name="theme-variant" enum="org.gnome.Terminal.ThemeVariant">
-      <default>'system'</default>
+      <default>'dark'</default>
       <summary>Which theme variant to use</summary>
     </key>
 
-- 
2.25.1

